### PR TITLE
Issue/2919 Accordion alignment of slotted heading with custom padding

### PR DESCRIPTION
**Pull request by tatjana-gartner2**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/issue/2919/

#### Scope

Short description of the pull request scope

#### Resolved Issue

Resolves [#2919]

### CODE REVIEW

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
   ([#2898](https://github.com/porsche-design-system/porsche-design-system/pull/2898))
 
 
+#### Fixed
+
+- `Accordion`: Alignment of slotted heading with custom padding ([2920](https://github.com/porsche-design-system/porsche-design-system/pull/2920))
```
Comment (tatjana-gartner2) [inline packages/components/CHANGELOG.md:35]:
```suggestion
- `Accordion`: Alignment of slotted heading with custom padding ([#2920](https://github.com/porsche-design-system/porsche-design-system/pull/2920))
```

File: packages/components-js/src/pages/accordion.html [hunk 2]
```diff
@@ -8,6 +8,12 @@
   </p-accordion>
 </div>
 
+<div class="playground light" title="should render single accordion with slotted heading with custom padding">
```
Comment (tatjana-gartner2) [inline packages/components-js/src/pages/accordion.html:11]:
```suggestion
<div class="playground light" title="should render single accordion with slotted heading and custom padding">
```

File: packages/components/src/components/accordion/accordion-styles.ts [hunk 3]
```diff
@@ -58,6 +58,7 @@ export const getComponentCss = (
         display: 'flex',
         position: 'relative',
         justifyContent: 'space-between',
+        alignItems: 'center',
```
Comment (denyo) [inline packages/components/src/components/accordion/accordion-styles.ts:61]: While this fixes the the alignment of the icons, isn't the different width still a problem?

Bildschirmfoto 2023-12-06 um 09 10 20

File: packages/components/CHANGELOG.md [hunk 4]
```diff
@@ -14,6 +14,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### [Unreleased]
 
+
+#### Added
+
+- `Accordion`: Custom clickarea for `compact`varaint ([#2920](https://github.com/porsche-design-system/porsche-design-system/pull/2920))
```
Comment (henrijoss) [inline packages/components/CHANGELOG.md:20]:
```suggestion
- `Accordion`: Custom clickarea for `compact` variant ([#2920](https://github.com/porsche-design-system/porsche-design-system/pull/2920))
```

**Comment by denyo** [comment]
Is this new hover style intentional?

Bildschirmfoto 2023-12-18 um 11 05 33

**Comment by denyo** [comment]
> Is this new hover style intentional?
> 
> Bildschirmfoto 2023-12-18 um 11 05 33

Maybe this should be covered via states VRT.

**Review by denyo — APPROVED** [review]

**Pull request merged**